server/report/report-service.js

```diff
@@ -102,4 +104,32 @@ export default class {
       path: path.join(options.path, filename)
     });
   }
+  remapSourceMaps(options) {
+    const remapIstanbul = Npm.require('remap-istanbul');
```

Comment:
@thiagodelgado111 If you mean `import remapIstanbul from 'remap-istanbul'`, it doesn't work.